### PR TITLE
Add Adopter to adopters list

### PR DESCRIPTION
**Pull request by DJDANNY123**
Signed-off-by: Danny Jackson <danny.jackson@surevine.com>

## Hey, I just made a Pull Request!
Adds Surevine into the list of adopters!

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
Hi! Happy to see you here!

As you may have seen the bot mention, we require that commits are DCO signed. This is an unfortunate but necessary impracticality in the open source repository. The action details have some explanatory text that may help to get that sorted. https://github.com/backstage/backstage/pull/9624/checks?check_run_id=5235170495

Don't hesitate to ask if there's something we can help with.

File: ADOPTERS.md [hunk 1]
```diff
@@ -93,4 +93,5 @@
...
\ No newline at end of file
+| [Bradesco](https://banco.bradesco/html/classic/sobre/index.shtm)                         | [@danilosoarescardoso](https://github.com/danilosoarescardoso)                                                                                                           | Centralized developer portal with software catalog, software templates, techdocs and some plugins from community and by our own.                                                                                                                                                                                                                                        |
+| [Surevine](https://www.surevine.com/)                         | [@DJDANNY123](https://github.com/DJDANNY123)                                                                                                           | Centralized oftware catalog for improving the way software is browsed, adopted and shared.                                                                                                                                                                                                                                         |
```
Comment (pjungermann) [inline ADOPTERS.md:97]: typo/missing "s" at software

**Comment by benjdlambert** [comment]
hey @DJDANNY123! we'd love to get this in but it looks like there's a conflict now too. If you need any help feel free to reach out!

**Comment by DJDANNY123** [comment]
Apologies, for leaving this open/unaddressed and cluttering up your PR list, I did attempt to include the sign off message in my commits, but I'll close this off and try again

**Pull request closed without merging**